docs/detections/rules-ui-create.asciidoc

```diff
@@ -24,7 +24,11 @@ To create a new detection rule, follow these steps:
 * You'll also need permissions to enable and view detections, manage rules, manage alerts, and preview rules. These permissions depend on the user role. Refer to <<detections-permissions-section>> for more information.
 --
 
-TIP: At any step, you can <<preview-rules,preview the rule>> before saving it to see what kind of results you can expect.
+[TIP]
+==============
+* At any step, you can <<preview-rules,preview the rule>> before saving it to see what kind of results you can expect.
+* To ensure rules don't search cold and frozen data when executing, configure the `excludedDataTiersForRuleExecution` advanced setting, or add a {ref}/sql-rest-filtering.html[Query DSL] filter to individual rules. Refer to <<exclude-cold-frozen-data-rule-executions>> to learn more.
```

Comment:
Drawing a little more distinction between the two use cases (my language is kinda clunky, feel free to tinker):
```suggestion
* To ensure rules don't search cold and frozen data when executing, either configure the `excludedDataTiersForRuleExecution` advanced setting (which applies to all rules in a space), or add a {ref}/sql-rest-filtering.html[Query DSL] filter to individual rules. Refer to <<exclude-cold-frozen-data-rule-executions>> to learn more.
```

Reply:
Applied in https://github.com/elastic/security-docs/commit/4a6659780a395f7fc6dd7e3400316bb7616ba1ba.